eclipse-collections/src/main/java/org/eclipse/collections/impl/collection/mutable/AbstractMultiReaderMutableCollection.java

```diff
@@ -78,231 +79,139 @@
  */
 public abstract class AbstractMultiReaderMutableCollection<T> implements MutableCollection<T>
 {
-    protected abstract MutableCollection<T> getDelegate();
-
-    protected abstract ReadWriteLock getLock();
-
-    protected void acquireWriteLock()
-    {
-        this.getLock().writeLock().lock();
-    }
+    protected transient ReadWriteLock lock;
+    protected transient ReadWriteLockWrapper lockWrapper;
 
-    protected void unlockWriteLock()
-    {
-        this.getLock().writeLock().unlock();
-    }
-
-    protected void acquireReadLock()
-    {
-        this.getLock().readLock().lock();
-    }
+    protected abstract MutableCollection<T> getDelegate();
 
-    protected void unlockReadLock()
+    protected ReadWriteLock getLock()
```

Comment:
do we need this method?

Reply:
I'll check for usages and remove if not used.

Reply:
removed